app/etc/modules/Mage_Captcha.xml

```diff
@@ -27,9 +27,7 @@
             <codePool>core</codePool>
             <depends>
                 <Mage_Adminhtml/>
-                <Mage_Customer/>
-                <Mage_Sendfriend/>
-                <Mage_Wishlist/>
+                <Mage_Customer/> 
```

Comment:
is the change to this file necessary for this PR?

Reply:
if we want the module of sendfriend/wishlist independent to other modules we have to modify this file. otherwise we can't remove wishlist/sendfriend module.

Reply:
and I checked the captcha using event and not deep bind to other modules. thus we can remove sendfriend and wishlist.

Reply:
Like mentioned before, I suggest tackling these changes one by one for ease of review.